octobot_trading/exchanges/rest_exchange.py

```diff
@@ -325,17 +328,23 @@ def get_uniform_timestamp(self, timestamp):
     def get_pair_from_exchange(self, pair) -> str:
         try:
             return self.client.market(pair)["symbol"]
-        except ccxt.base.errors.BadSymbol:
-            self.logger.error(f"Failed to get market of {pair}")
-            return None
+        except BadSymbol:
```

Comment:
No import required ?

Reply:
Already done [here](https://github.com/Drakkar-Software/OctoBot-Trading/pull/106/files/17ca52dbe0bc55b7b3c392b2301d8267127aa106..60d93597a36382259a99680f1cd578bce24aec71#diff-29e2db0b0f5a93571a9f481b0c2fe4a9R21)

Reply:
Oh, right, I didn't see it